encode_test.go

```diff
@@ -376,11 +376,11 @@ func TestParseRawContentOptionTrue(t *testing.T) {
 	if err := e.Root.Encode(b); err != nil {
 		t.Fatal(err)
 	}
-	test.DiffGolden(t, b.Bytes(), "testdata", "encode", "parser-raw-content-option-true.raw.golden")
+	test.DiffGolden(t, b.Bytes(), "testdata", "encode", "parser-raw-content-html-option-true.raw.golden")
```

Comment:
Could you add a comment above each of the four TestParseRawContentXXXXX test.DiffGolden calls stating what the test is verifying/expecting?  It won't be obvious to someone that may have to regenerate the golden files in the future.

Reply:
Hi, no problem. I've added the comments